### PR TITLE
create Responsive framework file 

### PR DESCRIPTION
**Pull request by RAWAN3830**
create Responsive framework file , if any changes happen then change  file properties by your self , team.

### CODE REVIEW

**Review by Darshan6069 — COMMENTED** [review]
Hello Ved added your main.dart file with changes

File: lib/core/Theme_data/app_text_theme.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class AppTextTheme {
+  static const mobileTextTheme = TextTheme(
+    headlineMedium: TextStyle(fontSize: 18, fontWeight: FontWeight.normal,),
```
Comment (nikkielevancex) [inline lib/core/Theme_data/app_text_theme.dart]: **suggestion**: create a base text style and then do copywith for all so you can avoid creating infinite text styles.

  static final baseTextStyle = TextStyle(
    fontFamily: FontFamily.signifyRelative,
    fontWeight: AppFontWeight.book,
    color: _defaultTextColor,
    letterSpacing: 0,
  );

File: lib/core/Theme_data/app_text_theme.dart [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    displayLarge: TextStyle(fontSize: 36, fontWeight: FontWeight.bold,),
+    displayMedium: TextStyle(fontSize: 28, fontWeight: FontWeight.bold,),
+    displaySmall: TextStyle(fontSize: 24, fontWeight: FontWeight.normal,),
+    bodyLarge: TextStyle(fontSize: 18, fontWeight: FontWeight.normal,),
+    bodyMedium: TextStyle(fontSize: 16, fontWeight: FontWeight.normal,),
+    bodySmall: TextStyle(fontSize: 14, fontWeight: FontWeight.normal,)
+  );
+
+  static const tabletTextTheme = TextTheme(
+      headlineMedium: TextStyle(fontSize: 24, fontWeight: FontWeight.normal,),
```
Comment (nikkielevancex) [inline lib/core/Theme_data/app_text_theme.dart]: **suggestion**: take font size from figma typography

File: lib/core/Theme_data/app_theme.dart [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+class AppTheme{
+  static ThemeData mobileThemeData = ThemeData(
+      textTheme: AppTextTheme.mobileTextTheme,
+      colorScheme: ColorScheme.fromSeed(seedColor:ThemeColor.mainColor),
+      useMaterial3: true
+  );
+
+  static ThemeData tabletThemeData = ThemeData(
+      textTheme: AppTextTheme.tabletTextTheme,
+      colorScheme: ColorScheme.fromSeed(seedColor: Colors.pinkAccent),
```
Comment (nikkielevancex) [inline lib/core/Theme_data/app_theme.dart]: **suggestion**: update colorscheme

File: lib/main.dart [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
-      home: const OnboardingScreen(),
+    return ResponsiveBreakpoints.builder(
+      child: Builder(
+          builder: (context) {
+            return MaterialApp(
+              title: 'Food Ninja',
+              theme: ThemeData(
+                colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+                useMaterial3: true,
+              ),
```
Comment (nikkielevancex) [inline lib/main.dart]: **suggestion**: take theme from appthemes

**Review by Darshan6069 — APPROVED** [review]

**Pull request merged**